ctapipe/tools/muon_reconstruction.py

```diff
@@ -24,65 +23,43 @@ class MuonAnalysis(Tool):
     name = "ctapipe-reconstruct-muons"
     description = traits.Unicode(__doc__)
 
-    output = traits.Path(directory_ok=False, help="HDF5 output file name").tag(
-        config=True
-    )
-
-    overwrite = traits.Bool(
-        default_value=False, help="If true, overwrite outputfile without asking"
-    ).tag(config=True)
-
     classes = [
         MuonProcessor,
         CameraCalibrator,
         TailcutsImageCleaner,
         EventSource,
         MuonRingFitter,
         MuonIntensityFitter,
+        DL1Writer,
     ]
 
     aliases = {
         "i": "EventSource.input_url",
         "input": "EventSource.input_url",
-        "o": "MuonAnalysis.output",
-        "output": "MuonAnalysis.output",
+        "o": "DL1Writer.output_path",
```

Comment:
I think the correct form of adding long / short forms is to use a tuple: `("i", "input")`, have to fix that for the other tools as well.